src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -441,13 +433,21 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
             env,
             digestUtil,
             ServerCapabilitiesRequirement.CACHE);
-      } catch (IOException e) {
-        if (verboseFailures) {
-          env.getReporter().handle(Event.warn(Throwables.getStackTraceAsString(e)));
-        }
-        env.getReporter().handle(Event
-            .warn("Failed to query remote cache capabilities: " + Utils.grpcAwareErrorMessage(e)));
+      }
+    } catch (IOException e) {
+      if (verboseFailures) {
+        env.getReporter().handle(Event.error(Throwables.getStackTraceAsString(e)));
+      }
+      String errorMessage =
+          "Failed to query remote execution capabilities: " + Utils.grpcAwareErrorMessage(e);
```

Comment:
You might want to also say how much time Bazel waited for the server to respond.

Reply:
Please see comment below.